pkgs/tools/admin/gixy/default.nix

```diff
@@ -0,0 +1,50 @@
+{ lib, fetchFromGitHub, python, pythonPackages
+, python27Packages, python35Packages, python36Packages }:
+let
+  pyPkgs = (if python.pythonVersion=="3.5"
+              then python35Packages else
+            if python.pythonVersion=="3.5"
+              then python36Packages
+            else
+              pythonPackages);
+in
+pyPkgs.buildPythonApplication rec {
+  name = "gixy-${version}";
+  version = "0.1.8";
+
+  # package is only compatible with python 2.7 and 3.5+
+  disabled = !(lib.versionAtLeast python.pythonVersion "3.5"
```

Comment:
pythonOlder or pythonNewer

Reply:
I didn't find `pythonNewer` anywhere and used `pythonAtLeast` instead, which I guess you meant anyway.